packages/cdktf-cli/lib/get/generator/provider-schema.ts

```diff
@@ -24,15 +28,68 @@ export interface Schema {
   block: Block;
 }
 
-export interface Attribute {
-  type: AttributeType;
+type AttributeNestedTypeNesting =
+  | "invalid"
+  | "single"
+  | "list"
+  | "set"
+  | "map"
+  | "group";
```

Comment:
`group` isn't valid for nested types, just regular types. I'm also not sure what it does is that case.

Reply:
You're right – I took that from the internal RFC but that does not seem to have made the cut to the implementation / or didn't make sense to implement 👍